src/lib.rs

```diff
@@ -33,6 +34,15 @@ const MASCOT: &[u8] = if cfg!(feature = "clippy") {
 // but also big enough for most inputs
 const BUFSIZE: usize = 2048;
 
+// moods
+
+#[derive(Copy, Debug, Clone, PartialEq, Eq, ValueEnum)]
```

Comment:
```suggestion
#[derive(Copy, Debug, Clone, PartialEq, Eq, ValueEnum)]
```
or a doc comment (using `///`) explaining what this enum does, e.g. the fact that it impacts the way we draw ferris